core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -161,11 +161,12 @@ describe('Topic and Story viewer functionality', function() {
 
     // Signing up with the login button should redirect the user back to the
     // exploration.
-    const loginButton = element(by.css('.protractor-test-login-button'));
-    await action.click('Login button', loginButton);
-    const useManualNavigation = false;
+    const LOGIN_BUTTON = element(by.css('.protractor-test-login-button'));
```

Comment:
Can this be changed to `var loginButton`?

Reply:
Done!